test/unit/puppetclass_lookup_key_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```

Comment:
These files don't really serve any purpose, we can do without them for the moment.